app/forms/gobierto_admin/admin_form.rb

```diff
@@ -23,7 +23,8 @@ class AdminForm
 
     validates :name, :email, presence: true
     validates :email, format: { with: Admin::EMAIL_ADDRESS_REGEXP }
-    validates :password, presence: true, confirmation: true
+    validates :password, presence: true, if: Proc.new { |f| f.admin.new_record? }
+    validates :password, confirmation: true, if: Proc.new { |f|  f.password != f.admin.password }
```

Comment:
Hey, sorry for intruding here, but wanted to point out that the conditional password confirmation check may not be needed here. I mean that the default `confirmation` validation behavior is taking the field presence into account. Also, I'd say we can take advantage of the `new_record?` property which is already there to condition the password presence check.

Let me please add a couple of commits to discuss that further.